go/libkb/pipeowner_windows.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
```

Comment:
should we conditionally compile this on windows only?

also 2018 instead of 2015 in copyright notice

Reply:
The `_windows` part of the filename makes it conditionally compiled, but I don't mind adding both